html2asketch/nodeToSketchLayers.js

```diff
@@ -137,6 +158,25 @@ export default async function nodeToSketchLayers(node) {
 
   const leaf = new ShapeGroup({x, y, width, height});
   const isImage = node.nodeName === 'IMG' && node.attributes.src;
+  const isSVG = node.nodeName.toLowerCase() === 'svg';
+
+  if (isSVG) {
+    const pathData = {};
+    let BCRNode = node;
+
+    if (node.childNodes.length === 1) {
+      BCRNode = node.childNodes[0];
```

Comment:
@KimDal-hyeong I don't quite understand why you need to do it. Why `BCRNode` is not always equal to `node`?

Reply:
oh, I think I see why. Importer deforms SVG for some reason
<img width="585" alt="screen shot 2018-03-12 at 22 34 03" src="https://user-images.githubusercontent.com/985504/37310859-95985f72-2645-11e8-931b-bd77a780589e.png">